### ISSUE TITLE
Support arbitrary status for SWITCH_DEFAULTS

### ISSUE THREAD

**Issue by AgDude** [adamchainz/gargoyle#196]
I would like the ability to allow our engineers to fully configure a new switch in code.  It looks like the only missing piece is being able to set an arbitrary status.  We are just starting to use Gargoyle, but it looks like we will need to set any new switch to "is_active=False", then in each environment manually change it in the admin after the release is deployed.

Something like:
```
from gargoyle.constants import DISABLED, EXCLUDE, GLOBAL, INCLUDE, INHERIT, SELECTIVE
GARGOYLE_SWITCH_DEFAULTS = {
    'new_switch': {
      'initial_status': SELECTIVE,
      'label': 'New Switch',
      'description': 'When you want the newness',
    },
}
````

**Comment by adamchainz** [comment]
Could work - would you be interested in starting a PR?

**Comment by adamchainz** [comment]
Fixed in #200, #206 .

Released in version 1.4.0: https://pypi.org/project/gargoyle-yplan/1.4.0/

Thanks! 🎉
